### PR TITLE
JSON format overhaul

### PR DESCRIPTION
**Pull request by hdashnow**
delete width, normal, pathogenic and intermediate fields. Any extra allele size info moved to details
delete notes_t2t field
convert field names to lowercase
rename fields: normal_min -> benign_min, normal_max -> benign_max, repeatunitlen -> motif_len removed notes field and moved content to details or another field as appropriate delete Mechanism_source and move content into mechanism_detail add locus and disease tag fields. Most are empty lists for now. convert many fields to lists

See #44 
Replaces #57 

Note: will provide a detail spec/schema in a separate PR.

### CODE REVIEW

**Review by vincerubinetti — APPROVED** [review]

**Comment by vincerubinetti** [comment]
How do you feel about merging #49 first, then I rebase this PR onto that and also update the new app to use the new format?

File: data/STRchive-database.json [hunk 1]
```diff
@@ -2759,61 +4249,96 @@
...
+        "benign_motif_gene_orientation":[],
+        "unknown_motif_reference_orientation":[],
+        "unknown_motif_gene_orientation":[],
         "disease":"Oculopharyngeal muscular dystrophy",
         "gene":"PABPN1",
         "flank_motif":null,
         "locus_structure":"(GCN)*",
-        "Inheritance":"AD\/AR",
+        "inheritance":[
+            "AD/AR"
```
Comment (laurelhiatt) [inline data/STRchive-database.json:4273]: this seems like a reasonable place to expand into the list

**Review by laurelhiatt — APPROVED** [review]

File: data/STRchive-database.json (hunk 1 above)
Comment (vincerubinetti) [reply data/STRchive-database.json:4273]: Looks like there's one entry, `OPMD_PABPN1`, where the split was missed: ` "inheritance": ["AD/AR"],`

**Comment by laurelhiatt** [comment]
I think I fixed it just now in the main v2 branch, but maybe it didn’t update because I’m not Harriet?

**Comment by laurelhiatt** [comment]
I probably let GitHub lie to me about what was the right thing to do...

**Pull request merged**